src/native-stack/navigators/createNativeStackNavigator.tsx

```diff
@@ -35,6 +35,18 @@ function NativeStackNavigator({
     screenOptions,
   });
 
+  // Starting from React Navigation v6 @react-navigation/native-stack should be used.
```

Comment:
```suggestion
  // Starting from React Navigation v6, @react-navigation/native-stack should be used.
```

Reply:
FIxed in 7c22e558423fd3d66c242c23092e9288c8cf69a9